src/application/common/endpoint.ts

```diff
@@ -24,7 +27,21 @@ export class Endpoint {
     }
 
     protected get host() {
-        return location.host || "127.0.0.1:3000"
+
+
+        if (location.host === "") {
+            this.fs.readFile('port', "utf8", function (err: string, data: any) {
```

Comment:
@akosyakov The idea of the file is to use it in webpack also... or whatever the production proxy is... maybe we could use injection inside the app but still write it in one place for external clients?